docs/wiki/99_Developers/Reporting_Issues.md

```diff
@@ -0,0 +1,10 @@
+# Reporting Issues
```

Comment:
What does this have to do with a glossary?

Reply:
Hi @driusan, this was part of the wiki migration task, but was mixed in this PR. Thank you!

Reply:
This file is still here.